### PR TITLE
Docs/cdc documentation

### PR DESCRIPTION
**Pull request by Miles-Garnsey**
**What this PR does**:

Adds CDC docs.

**Which issue(s) this PR fixes**:
#k8ssandra-1455

**Checklist**
- [ ] Changes manually tested
- [ ] Automated Tests added/updated
- [ ] Documentation added/updated
- [ ] CHANGELOG.md updated (not required for documentation PRs)
- [ ] CLA Signed:  [DataStax CLA](https://cla.datastax.com/)

### CODE REVIEW

**Comment by Miles-Garnsey** [comment]
This needs to be tested as during the 1.2 release, `auth: false` started causing NPEs.

**Comment by Miles-Garnsey** [comment]
@adejanovski I've put this ticket in the k8ssandra-operator repo. Not sure if you wanted it in k8ssandra instead.

File: docs/enable-cdc.md
Comment (adejanovski) [inline docs/enable-cdc.md:24]:
```suggestion
### Deploy a K8ssandraCluster

Next, you'll want to deploy a K8ssandraCluster with the Pulsar service's location referenced:
```

File: docs/enable-cdc.md
Comment (adejanovski) [inline docs/enable-cdc.md:52]:
```suggestion
### Create your tables with cdc enabled

When creating your tables, you'll need to do so with `cdc=true` as below:
```

**Review by adejanovski — CHANGES_REQUESTED** [review]
Small changes required, and then good to go 👍

File: docs/enable-cdc.md
Comment (adejanovski) [inline docs/enable-cdc.md:24]:
```suggestion
### Deploy a K8ssandraCluster

Next, you'll want to deploy a K8ssandraCluster with the Pulsar service's location referenced:
```

**Review by adejanovski — APPROVED** [review]
This can be merged once the `K8ssandraDatacenter`  refs are replaced by `K8ssandraCluster` ones.
I'll port this over to the K8ssandra.io docs once it's done.

**Comment by Miles-Garnsey** [comment]
> This can be merged once the K8ssandraDatacenter  refs are replaced by K8ssandraCluster ones.
I'll port this over to the K8ssandra.io docs once it's done.

Is that true though, the particular part of the manifest we're talking about here is indeed the K8ssandraDatacenter (not the overall cluster). But then, we never really reference the term K8ssandraDatacenter in the YAML API do we... What do you think is clearer?

**Comment by adejanovski** [comment]
> > This can be merged once the K8ssandraDatacenter  refs are replaced by K8ssandraCluster ones.
> > I'll port this over to the K8ssandra.io docs once it's done.
> 
> Is that true though, the particular part of the manifest we're talking about here is indeed the K8ssandraDatacenter (not the overall cluster). But then, we never really reference the term K8ssandraDatacenter in the YAML API do we... What do you think is clearer?

We don't have any object named a `K8ssandraDatacenter` AFAIK 🤔 We create `K8ssandraCluster` objects which will create `CassandraDatacenter` ones. So maybe you meant `CassandraDatacenter` instead?

**Comment by Miles-Garnsey** [comment]
> We don't have any object named a K8ssandraDatacenter AFAIK thinking We create K8ssandraCluster objects which will create CassandraDatacenter ones. So maybe you meant CassandraDatacenter instead?

Right, that's true (and an ongoing gripe I have, but anyway...)

So should we say CassandraDatacenter or K8ssandraCluster then?

**Comment by adejanovski** [comment]
> So should we say CassandraDatacenter or K8ssandraCluster then?

Since this documentation is for k8ssandra-operator I think it would be less confusing to go with `K8ssandraCluster`.

**Comment by Miles-Garnsey** [comment]
All fixed (I think, some of these were a bit embarrassing) let me know if you're happy for me to merge.

I would like to test this against `main` however before merging, because keep in mind it hasn't been properly tested due to the auth bug.

**Review by adejanovski — APPROVED** [review]
Thanks for the updates 👍 

:shipit: 
emoji-icon-glossy-05-06-travel-places-transport-water-u-boat-72dpi-forPersonalUseOnly (1)

**Comment by Miles-Garnsey** [comment]
Made one or two tiny changes having just re-tested again. All looking good.

**Pull request merged**
